### DISCUSSION TITLE
how to quick change directory in nvim?

### DISCUSSION THREAD

**Discussion by jackywu** [ibhagwan/fzf-lua#1395] [Q&A] [answered]
I found there is a way to do it https://github.com/junegunn/fzf.vim/issues/251#issuecomment-311717789,
```
command! -nargs=* -complete=dir Cd call fzf#run(fzf#wrap(
  \ {'source': 'find '.(empty(<f-args>) ? '.' : <f-args>).' -type d',
  \  'sink': 'cd'}))
```

 but didn't found how to do that by this plugin. any helps?

**Comment by ibhagwan** [comment, marked as answer]
Search the issues and read the advanced Wiki and everything will be clear.
https://github.com/ibhagwan/fzf-lua/wiki/Advanced

What you're asking is very easy with the `fzf_exec` API, modify this to your liking (requires `fd`, change to `find` if you wish):
```lua
require("fzf-lua").fzf_exec("fd --type d . $HOME", {
  actions = {
    ["enter"] = {
      desc = "change-directory",
      fn = function(sel)
        vim.cmd("cd " .. sel[1])
      end
    }
  }
})
```

**Comment by ibhagwan** [comment]
Btw, to write this as a user command named `Cd` use the `vim.api.nvim_create_user_command` API, `:FzfLua helptags` is your friend.

**Reply by jackywu** [reply]
so nice，thanks a lot
